OpenSim/Simulation/Model/Frame.h

```diff
@@ -193,6 +206,41 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Frame, ModelComponent);
     */
     SimTK::Vec3 findLocationInAnotherFrame(const SimTK::State& state, 
                     const SimTK::Vec3& point_F, const Frame& otherFrame) const;
+
+    /**
+    Take a point located and expressed in this frame (F) and determine
+    its location expressed in Ground (G). This method is equivalent to
+    `findLocationInAnotherFrame()` where `otherFrame` is always Ground.
+
+    @param state       The state of the model.
+    @param point_F     The point to be re-expressed.
+    @return point_G    The re-expression of the point in Ground.
+    */
+    SimTK::Vec3 findLocationInGround(const SimTK::State& state,
+                    const SimTK::Vec3& point_F) const;
+
+    /**
+    Take a point located and expressed in this frame (F) and determine
+    its velocity expressed in Ground (G).
+
+    @param state       The state of the model.
+    @param point_F     The point to be re-expressed.
+    @return point_G    The re-expression of the point in Ground.
+    */
+    SimTK::Vec3 findVelocityInGround(const SimTK::State& state,
```

Comment:
IMHO I would expect the velocity of a Frame to be 6 numbers - a SpatialVec, so calling it SpatialVelocity is somewhat unnecessary and confusing. We are then using the Frame (it has a handy helper) to find a velocity of a point fixed to that Frame, so that method could indicate what it operates on. I am happy with *Station* as well. I think *Point* is also OK because these methods do not take either a `Station` or `Point` (`Component`) but just a `Vec3` which cannot be a free floating point. The comments seem clear/unambiguous with `point_F` and `point_G` and so *Point* seems more natural.